feat(content): Add a Pirate-world mission about extortion (#6161)
endless-sky/endless-sky: data/human/transport missions.txt

	on complete
		payment 30000
		dialog "As you descend through the upper atmosphere, you inform your skydivers that it's time to jump. However, after being able to see the distances and atmospheric conditions involved, all of them are far less keen on the idea than when you first picked them up. You bring them down to the landing pad, and they pay you a meager <payment> before hopping off."

> mission "Mafia Extortion"

Reviewer: ```suggestion
mission "Mafia Extortion"
	minor
```

	source
		government "Pirate"
	on accept

> 		payment -200000

Reviewer: ```suggestion
		payment -200000
		fail
```

	to offer
		random < 2
		credits >= 2000000
	on offer
		conversation
			`A man in a tailored suit, sunglasses, shiny shoes, and green hair approaches you. "Hey, that's a very nice ship you have there. Would be a shame if anything happened to it."`

> 			`You notice that he is missing a finger on each hand.`

Reviewer: ```suggestion
			`	You notice that he is missing a finger on each hand.`
```

			choice
				`"Thank you. My name is <first> <last>. To whom do I have the pleasure of talking to?"`
					goto nameless
				`"Yes, I'm very happy with my ship."`
					goto specifics
				`"What should I be afraid of happening to my ship?"`
					goto specifics
				`"You sound like one of those Mafia goons."`
					goto mafia
				`"Are you trying to extort me?"`
					goto mafia

> 				`"Yes, it helps me transport stuff for some of the most dangerous people on this planet."`

Reviewer 2: ```suggestion
				`"Yes, it helps me transport goods for some of the most dangerous people on this planet."`
```

					goto unnerved
				`	"I don't have time to talk."`
					goto funeral
			label nameless

> 			`	Your greeting gets a grumpy reply about the value of anonymity in a place like this.`

Reviewer: This part should probably be spoken aloud.
```suggestion
			`	The man snorts. "Wow, telling everybody you meet your full name. That totally won't end up painting a huge target on your head if said name has a bounty on it. But I digress."`
```

			choice

> 				`"Sounds like you are afraid of talking to me. Why is that?"`

Reviewer 3: ```suggestion
				`"Sounds like you're afraid of talking to me. Why is that?"`
```

					goto mafia
				`"What should I be afraid of happenning to my ship?"`
					goto specifics
				`"You sound like one of those mafia goons."`
					goto mafia
				`"Are you trying to extort me?"`
					goto mafia
			label specifics

> 			`The man responds, "Well, a fleet of unfriendly ships in this system might just blow your ship to pieces."`

Reviewer 2: ```suggestion
			`The man responds, "Well, a fleet of unfriendly ships in this system might blow you to pieces when you take off. It's been known to happen from time to time."`
```

			choice
				`"Ha, business as usual in this system."`
					goto unconcerned
				`"Are you trying to extort me?"`
					goto mafia
				`"And you have a way to prevent this from happening?"`
					goto mafia

> 				`"And you are the local goon that takes bribes to keep such ships docked?"`

Reviewer 2: ```suggestion
				`"And you're the local goon who takes bribes to keep such ships docked?"`
```

					goto mafia

> 				`"That would be unfortunate, I run jobs for some of the most dangerous people on this planet and they wouldn't be happy if that happened."`

Reviewer 2: ```suggestion
				`"That would be unfortunate. I run jobs for some of the most dangerous people on this planet and they wouldn't be happy if that happened."`
```

					goto unnerved
				`"I really don't have time to talk to you."`
					goto funeral
			label unconcerned
			`The man responds, "Maybe you want to improve that situation?`
				goto donation
			label mafia

> 			`The man acts as if he is shocked by your remark. "No way! I'm just a simple man walking around the spaceport and helping people out. You would be wise to listen to locals with experience, especially in dangerous places like this."`

Reviewer 2: ```suggestion
			`The man acts shocked by your remark. "No way! I'm just a simple man walking around the spaceport and helping people out. You would be wise to listen to locals with experience, especially in dangerous places like this."`
```

			label donation
			`	"If you care about safety, then I could make a donation to the System Safety Foundation on your behalf. 200,000 credits really goes a long way around here."`
			choice
				`(pay 200,000 credits)`
					goto payup
				`"How can I be sure the money is spent well?"`
					goto impatient
				`"Sorry, I already made a donation to the 'Security Support Fund' earlier today and I don't donate twice in one day."`
					goto impatient
				`"I've already heard enough, I'm going to talk to the authorities."`
					goto next_steps
				`"And what dangers should I be afraid of?"`
					goto impatient

> 				`"Listen, I see that you are missing a finger on each hand. You look like a goon, and not a successful one."`

Reviewer 2: ```suggestion
				`"Listen, I see that you're missing a finger on each hand. You look like a goon, and not a very successful one."`
```

					goto unnerved
			label impatient
			`The man sounds annoyed. "Listen, I don't have all day to talk! Are you going to pay or not?!"`
			choice
				`(pay 200,000 credits)`
					goto payup
				`"I don't trust you. You aren't getting any money from me."`
					goto funeral
				`"I've heard enough, I'm going to talk to the authorities."`
					goto next_steps

> 				`"Listen, I see that you are missing a finger on each hand. You look like a goon, and not a successful one."`

Reviewer 2: ```suggestion
				`"Listen, I see that you're missing a finger on each hand. You look like a goon, and not a very successful one."`
```

					goto unnerved
			label funeral
			`	The man responds "Your funeral, not mine." and turns away from you.`
				goto walks_away
			label next_steps
			`	"Look, that's not a smart move. Most of them are in on it anyway."`
			label walks_away
			`	The man walks away. You consider what to do next.`
			choice
				`	(Shrug off the whole thing and take no further action.)`
					decline
				`	(Find the closest thing to a police force that this planet has.)`
					goto police
				`	(Go to a bar and ask for whoever is in charge.)`
					goto bar
			label police

> 			`It's not long before you find the ruins of what used to be a police station... you think. The building has been thoroughly destroyed, hopefully not while anybody was present.`

Reviewer: ```suggestion
			`	It's not long before you find the ruins of what used to be a police station... you think. The building has been thoroughly destroyed, hopefully not while anybody was present.`
```

			choice
				`	(Go to a bar and ask for whoever is in charge.)`
					goto bar
				`	(Shrug off the whole thing and take no further action.)`
					decline
			label bar

> 			`You walk towards the closest bar, but stumble into the green-haired man that you just had a conversation with.`

Reviewer: ```suggestion
			`	You walk towards the closest bar, but stumble into the green-haired man that you just had a conversation with.`
```

> 			`He looks surprised and asks, "Were you going to rat me out to the local boss?"`

Reviewer: ```suggestion
			`	He looks surprised and asks, "Were you going to rat me out to the local boss?"`
```

			choice
				`"What an excellent idea! Yes, I'll just do that."`
				`"What I'm going to do is none of your business."`
				`"Don't worry, I'm just taking a walk."`
			label unnerved

> 			`The man is starting to act a bit nervous. "Sorry I bothered you. Maybe let's forget it happened?"`

Reviewer 3: ```suggestion
			`The man is starting to act a bit nervous. "Sorry I bothered you. Can we forget this happened?"`
```

> 			`The man is starting to act a bit nervous. "Sorry I bothered you. Can we forget this happened?"`

Reviewer: ```suggestion
			`	The man is starting to act a bit nervous. "Sorry I bothered you. Can we forget this happened?"`
```

			choice
				`	"Well, conversations like this are hard to forget."`
					goto cornered
				`	"Nice scam you have running here. Would be a shame if somebody told the local godfather about it."`
					goto payback
				`	(Agree.)`
					goto zerodeal
			label cornered
			`	"Well what do you want? Is there anything I can do for you?"`
			choice
				`	"No, let's just forget it happened."`
					goto zerodeal

> 				`	"Yes, what an excellent idea. How much do you want to pay me?"`

Reviewer: ```suggestion
				`	"Compensation for the time that you've wasted would be excellent."`
```

			label payback

> 			`The man is now really nervous and asks: "How many credits do you need to just forget this whole conversation happened?"`

Reviewer: ```suggestion
			`	A bead of sweat travels down the man's face as he asks, "How many credits do you need to just forget this whole conversation happened?"`
```

			choice

> 				`	"100,000,000"`

Reviewer: ```suggestion
				`	"100,000,000."`
```

					goto far_too_much

> 				`	"1,000,000"`

Reviewer: ```suggestion
				`	"1,000,000."`
```

					goto too_much

> 				`	"200,000"`

Reviewer: ```suggestion
				`	"200,000."`
```

					goto too_much

> 				`	"50,000"`

Reviewer: ```suggestion
				`	"50,000."`
```

					goto too_much

> 				`	"5,000"`

Reviewer: ```suggestion
				`	"5,000."`
```

					goto counterdeal

> 				`	"2,000"`

Reviewer: ```suggestion
				`	"2,000."`
```

					goto counterdeal

> 				`	"1,000"`

Reviewer: ```suggestion
				`	"1,000."`
```

					goto counterdeal
				`"I don't need your credits, just forget it."`
					goto zerodeal
			label zerodeal
			`The man walks away without saying a word.`
				decline
			label too_much

> 			`"That is too much! 2389 credits is all I have. Is that sufficient for you?"`

Reviewer 2: ```suggestion
			`"That's too much! 2389 credits is all I have. Would that be enough?"`
```

			choice

> 				`	"Sure"`

Reviewer: ```suggestion
				`	"Sure."`
```

					goto counterdeal
				`"No way, I want a lot more."`
					goto far_too_much
			label counterdeal
			`The man hands you a credit chip, nods and walks away. You pocket the chip quickly, more ready to be done with this neighborhood than you want to visibly show.`
			`A few minutes later you reach into your pocket for the chip... to find it missing.`

> 			`Looks like even a bad-ass captain like you isn't immune to pickpocketing on a planet like this.`

Reviewer 2: ```suggestion
			`Looks like even a hardened captain like you isn't immune to being pickpocketed on a planet like this.`
```

> 			`A few minutes later you reach into your pocket for the chip... to find it missing.`
> 			`	Looks like even a hardened captain like you isn't immune to being pickpocketed on a planet like this.`

Reviewer: ```suggestion
			`	A few minutes later you reach into your pocket for the chip... to find it missing. Looks like even a hardened captain like you isn't immune to being pickpocketed on a planet like this.`
```

				decline
			label far_too_much

> 			`The man starts panicking and is clumsily reaching in his jacket, maybe to get some money, but more likely reaching for a gun.`

Reviewer: ```suggestion
			`	The man starts panicking and clumsily reaches into his jacket, maybe to get some money, but more likely reaching for a gun.`
```

			choice
				`	(Try to calm down the man.)`
					goto draws
				`	(Mock the man further.)`
					goto draws
				`	(Draw your sidearm and shoot the man.)`
					goto shoot
			label draws

> 			`	The man somehow manages to draw a weapon and tries to shoot at you, but the weapon doesn't seem to function.`

Reviewer: ```suggestion
			`	The man manages to draw a weapon! You brace for impact, but after a few seconds of waiting, you realize his weapon is only making a clicking sound when he tries to shoot.`
```

			choice

> 				`	"Ha ha, you are really a failure as a goon."`

Reviewer: ```suggestion
				`	"Ha, ha, you really are a failure of a goon."`
```

				`	(Draw your sidearm and shoot the man.)`
					goto shoot

> 			`The man tries to fire his weapon once more, but now it does function and the bullet hits you between the eyes. You die almost instantly, but you do have a few microseconds to reflect on the wisdom of mocking a man that is trying to fire a gun at you.`

Reviewer: ```suggestion
			`	As you finish talking, you see a flash of light come from the muzzle of the man's weapon as a bullet hits you between the eyes. You die almost instantly, but you do have a few microseconds to reflect on the wisdom of mocking a man that is trying to fire a gun at you.`
```

				die
			label shoot

> 			`You shoot and hit the man between the eyes; he dies instantly and a crowd starts to form around you.`

Reviewer 2: ```suggestion
			`Your bullet hits the man right between the eyes; he dies instantly, and a crowd starts to form around you.`
```

> 			`Most bystanders are surprised, but you also notice an angry sentiment brewing in the (not unarmed) crowd. It might be wise to leave quickly.`

Reviewer: ```suggestion
			`	Most bystanders are surprised, but you also notice an angry sentiment brewing in the (not unarmed) crowd. It might be wise to leave quickly.`
```

			choice
				`	(Run towards your ship and leave immediately.)`
					goto leave
				`	(Put away your sidearm and pretend nothing happened.)`

> 			`You walk away nonchalantly, but you don't get far before multiple laser blasts slice through you.`

Reviewer: ```suggestion
			`	You walk away nonchalantly, but you don't get far before multiple laser blasts slice through you.`
```

				die
			label leave

> 			`	You make it to your ship alive and depart under a rain of bullets and laser-weapons fire. This is a pirate planet so there are unlikely to be any lasting consequences, but you might want to stay away from for a few days.`

Reviewer 4: ```suggestion
			`	You make it to your ship alive and depart under a rain of bullets and laser-weapons fire. This is a pirate planet so there are unlikely to be any lasting consequences, but you might want to stay away for a few days.`
```

				flee
			label payup

> 			`The man takes your credits, responds with a brief "Thank you," and walks away.`

Reviewer: ```suggestion
			`	The man takes your credits, responds with a brief "Thank you," and walks away.`
```

	on complete
		payment 30000
		dialog "As you descend through the upper atmosphere, you inform your skydivers that it's time to jump. However, after being able to see the distances and atmospheric conditions involved, all of them are far less keen on the idea than when you first picked them up. You bring them down to the landing pad, and they pay you a meager <payment> before hopping off."

mission "Mafia Extortion"
	source
		government "Pirate"
	on accept
		payment -200000
	to offer
		random < 2
		credits >= 2000000
	on offer
		conversation

> 			`A man in a tailored suit, sunglasses and shiny shoes with green hair approaches you. "Hey, that's a very nice ship you have there. Would be a shame if anything happened to it."`

Reviewer 3: ```suggestion
			`A man in a tailored suit, sunglasses, shiny shoes, and green hair approaches you. "Hey, that's a very nice ship you have there. Would be a shame if anything happened to it."`
```

			`You notice that he is missing a finger on each hand.`
			choice

> 				`"Thank you. My name is <first> <last>. Whom do I have the pleasure of talking to?"`

Reviewer 3: ```suggestion
				`"Thank you. My name is <first> <last>. To whom do I have the pleasure of talking to?"`
```

Reviewer 5: Double "to" is weird here, I'd get rid of one or the other.

			choice

> 				`	"Thank you. My name is <first> <last>. To whom do I have the pleasure of talking to?"`

Reviewer 2: ```suggestion
				`	"Thank you. My name is <first> <last>. To whom do I have the pleasure of speaking?`
```

(Otherwise the verb has two instances of "to" that apply to it)

					goto nameless
				`	"Yes, I'm very happy with my ship."`
					goto specifics
				`	"What should I be afraid of happening to my ship?"`
					goto specifics
				`	"You sound like one of those Mafia goons."`
					goto mafia
				`	"Are you trying to extort me?"`
					goto mafia
				`	"Yes, it helps me transport goods for some of the most dangerous people on this planet."`
					goto unnerved
				`	"I don't have time to talk."`
					goto funeral
			label nameless
			`	The man snorts. "Wow, telling everybody you meet your full name. That totally won't end up painting a huge target on your head if said name has a bounty on it. But I digress."`
			choice

> 				`	"Sounds like you're afraid of talking to me. Why is that?"`

Reviewer 2: Hmm, I'm not sure the above line of dialogue gives the impression that the dude is afraid of you. It seems like he's dissing you. So this response option doesn't make sense to me.

Author: Yes, it worked better with the grumpy reply that we had on line 3339 before. I'll remove this choice.

					goto mafia
				`	"What should I be afraid of happening to my ship?"`
					goto specifics
				`	"You sound like one of those mafia goons."`
					goto mafia
				`	"Are you trying to extort me?"`
					goto mafia
			label specifics
			`	The man responds, "Well, a fleet of unfriendly ships in this system might blow you to pieces when you take off. It's been known to happen from time to time."`
			choice

> 				`	"Ha, business as usual in this system."`

Reviewer 2: ```suggestion
				`	"Ha, business as usual around here."`
```

(the player echoing "in this system" sounds odd to me)

					goto mafia
				`"And you have a way to prevent this from happening?"`
					goto mafia
				`"And you are the local goon that takes bribes to keep such ships docked?"`
					goto mafia
				`"That would be unfortunate, I run jobs for some of the most dangerous people on this planet and they wouldn't be happy if that happened."`
					goto unnerved
				`"I really don't have time to talk to you."`
					goto funeral
			label unconcerned
			`The man responds, "Maybe you want to improve that situation?`
				goto donation
			label mafia

> 			`The man acts as if he is shocked by your remark. "No way! I'm just a simple man walking around the spaceport and talking about life. You would be wise to listen to locals with experience, especially in dangerous places like this."`

Reviewer 3: I feel like this would fit better, but feel free to ignore.
```suggestion
			`The man acts as if he is shocked by your remark. "No way! I'm just a simple man walking around the spaceport and helping people out. You would be wise to listen to locals with experience, especially in dangerous places like this."`
```

			label donation
			`	"If you care about safety, then I could make a donation to the System Safety Foundation on your behalf. 200,000 credits really goes a long way around here."`
			choice
				`(pay 200,000 credits)`
					#payment -200000

> 					goto next_steps

Reviewer 3: Did you meant for this choice to go to "next_steps"?

Author: No, I'll update this to go to `payup`.

				`"How can I be sure the money is spent well?"`
					goto impatient
				`"Sorry, I already made a donation to the 'Security Support Fund' earlier today and I don't donate twice in one day."`
					goto impatient
				`"I've already heard enough, I'm going to talk to the authorities."`
					goto next_steps
				`"And what dangers should I be afraid of?"`
					goto impatient
				`"Listen, I see that you are missing a finger on each hand. You look like a goon, and not a successful one."`
					goto unnerved
			label impatient
			`The man sounds annoyed. "Listen, I don't have all day to talk! Are you going to pay or not?!"`
			choice
				`(pay 200,000 credits)`
					goto payup

> 				`"I don't trust you, you aren't getting any money from me."`

Reviewer 3: ```suggestion
				`"I don't trust you. You aren't getting any money from me."`
```

					goto funeral
				`"I've heard enough, I'm going to talk to the authorities."`
					goto next_steps
				`"Listen, I see that you are missing a finger on each hand. You look like a goon, and not a successful one."`
					goto unnerved
			label funeral
			`The man responds "Your funeral, not mine." and turns away from you.`
				goto walks_away
			label next_steps
			`"Look, that's not a smart move. Most of them are in on it anyway."`
			label walks_away
			`The man walks away. You consider what to do next.`
			choice
				`(Shrug off the whole thing and take no further action.)`
					decline

> 				`(Find the closest thing to police force that this planet has.)`

Reviewer 3: ```suggestion
				`(Find the closest thing to a police force that this planet has.)`
```

					goto police
				`(Go to a bar and ask for whoever is in charge.)`
					goto bar
			label police
			`It's not long before you find the ruins of what used to be a police station... you think. The building has been thoroughly destroyed, hopefully not while anybody was present.`
			choice
				`(Go to a bar and ask for whoever is in charge.)`
					goto bar
				`(Shrug off the whole thing and take no further action.)`

> 					accept

Author: Good catch, should be a `decline`

> 			`	It's not long before you find the ruins of what used to be a police station... you think. The building has been thoroughly destroyed, hopefully not while anybody was present.`

Reviewer 2: ```suggestion
			`	It's not long before you find the ruins of what used to be a police station... you think. The building has been thoroughly destroyed, hopefully not while anybody was inside.`
```

			label bar
			`You walk towards the closest bar, but stumble into the green-haired man that you just had a conversation with.`
			`He looks surprised and asks, "Were you going to rat me out to the local boss?"`
			choice
				`"What an excellent idea! Yes, I'll just do that."`
				`"What I'm going to do is none of your business."`
				`"Don't worry, I'm just taking a walk."`
			label unnerved
			`The man is starting to act a bit nervous. "Sorry I bothered you. Maybe let's forget it happened?"`
			choice
				`"Well, conversations like this are hard to forget."`
					goto cornered

> 				`"Nice scam you have running here, would be a shame if somebody would tell the local godfather about it."`

Reviewer 3: ```suggestion
				`"Nice scam you have running here. Would be a shame if somebody told the local godfather about it."`
```

					goto payback
				`(Agree)`
					goto zerodeal
			label cornered

> 			`"Well what do you want, is there anything I can do for you?"`

Reviewer 3: ```suggestion
			`"Well what do you want? Is there anything I can do for you?"`
```

> 			`	"Well what do you want? Is there anything I can do for you?"`

Author: "Hey, that's a very nice ship you have there. Would be a shame if anything happened to it." is usually also spoken very politely. But you are right, the conversation would be less polite at this point.

			choice
				`	"No, let's just forget it happened."`
					goto zerodeal
				`	"Compensation for the time that you've wasted would be excellent."`
			label payback
			`	A bead of sweat travels down the man's face as he asks, "How many credits do you need to just forget this whole conversation happened?"`
			choice
				`	"100,000,000."`
					goto far_too_much
				`	"1,000,000."`
					goto too_much
				`	"200,000."`
					goto too_much
				`	"50,000."`
					goto too_much
				`	"5,000."`
					goto counterdeal
				`	"2,000."`
					goto counterdeal
				`	"1,000."`
					goto counterdeal
				`	"I don't need your credits, just forget it."`
					goto zerodeal
			label zerodeal
			`The man walks away without saying a word.`
				decline
			label too_much

> 			`	"That's too much! 2389 credits is all I have. Would that be enough?"`

Reviewer 6: ```suggestion
			`	"That's too much! 2,389 credits is all I have. Would that be enough?"`
```
I _think_ credits usually have commas. Don't quote me though

				`"Sure"`
					goto counterdeal
				`"No way, I want a lot more."`
					goto far_too_much
			label counterdeal
			`The man hands you a credit chip, nods and walks away. You pocket the chip quickly, more ready to be done with this neighborhood than you want to visibly show.`
			`A few minutes later you reach into your pocket for the chip... to find it missing.`

> 			`Looks like even a bad-ass captain like you is not immune to pickpocketing on a planet like this.`

Reviewer 3: ```suggestion
			`Looks like even a bad-ass captain like you isn't immune to pickpocketing on a planet like this.`
```
